src/sctptransport.cpp

```diff
@@ -445,6 +445,7 @@ int SctpTransport::handleWrite(byte *data, size_t len, uint8_t tos, uint8_t set_
 		mWritten = true;
 		mWrittenOnce = true;
 		mWrittenCondition.notify_all();
+		mBytesSent += len;
```

Comment:
Actually, this will count everything SCTP outputs, counting control, retransmissions, etc.

I think the incrementation should be done when sent messages are successfully passed to SCTP instead, at the end of `trySendMessage` (when it returns `true`). Of course it should be incremented only if the message type is `String` or `Binary`.

Reply:
I am trying p2p test app.
offerer output
```
Bytes Sent:1504 Bytes Received:0 Round-Trip Time:30 ms
```

answerer output
```
Bytes Sent:1388 Bytes Received:3 Round-Trip Time:2 ms
```

I sent a message from offerer to answerer ( 3 chars)

Reply:
And it counts only messages that I sent (adds message char count)

Reply:
OK. My fault. I was looking wrong point. 
I corrected all. Now only counting user messages.